tests/remote/test_fennec.py

```diff
@@ -17,7 +17,8 @@
     # -p android-api-9 --branch=mozilla-central
     {'args': {'application': 'fennec',
               'platform': 'android-api-9',
-              'branch': 'mozilla-central'}},
+              'branch': 'mozilla-central',
+              'date': '2016-01-01'}},
```

Comment:
The last one I found was 2016-03-11. I changed the date to that now. Should go through fine now...